oxidation/crates/parsec_api_protocol/src/cmds.rs

```diff
@@ -0,0 +1,202 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+macro_rules! cmds_bundle {
+    ($cmds: ident, [$(($path: path, $module: ident, $cmd: ident $(, $rename: literal)? $(,)?),)*]) => {
+
+        ::paste::paste! {
+            pub mod $cmds {
+                use serde::{Deserialize, Serialize};
```

Comment:
Better use `::serde::...` (so with leading :: ) to ensure the path is absolute.

Here it is not useful given the macro is private, but I think it is better to always write it this way to avoid copy/paste or refactoring errors

Reply:
and same thing for use of `rpm_serde::`

Reply:
Done :+1: